exporters/otlp/otlplog/otlploggrpc/client.go

```diff
@@ -132,25 +158,41 @@ func (c *client) UploadLogs(ctx context.Context, rl []*logpb.ResourceLogs) error
 	ctx, cancel := c.exportContext(ctx)
 	defer cancel()
 
-	return c.requestFunc(ctx, func(ctx context.Context) error {
+	trackExportFunc := c.trackExport(context.Background(), int64(len(rl)))
+	err := c.requestFunc(ctx, func(ctx context.Context) error {
 		resp, err := c.lsc.Export(ctx, &collogpb.ExportLogsServiceRequest{
 			ResourceLogs: rl,
 		})
+
 		if resp != nil && resp.PartialSuccess != nil {
 			msg := resp.PartialSuccess.GetErrorMessage()
 			n := resp.PartialSuccess.GetRejectedLogRecords()
 			if n != 0 || msg != "" {
 				err := fmt.Errorf("OTLP partial success: %s (%d log records rejected)", msg, n)
+				trackExportFunc(err, codes.OK)
```

Comment:
This relies on manual understanding that this needs to be called for each return path of the function. Use `defer` and any other variables instead to ensure that future changes do not inadvertently introduce un-handled return paths.